### PR TITLE
Play list management

### PR DESCRIPTION
**Pull request by SadiqZak**
Added functionality to play list.

### CODE REVIEW

**Comment by SadiqZak** [comment]
playlistmanagement

File: src/Backend/controllers/filter.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    :type==="Saga"
+    ? arr.filter((item)=>item.category==="Saga")
+    :type==="Comedy"
+    ? arr.filter((item)=>item.category==="Comedy")
+    :type==="Podcasts"
+    ? arr.filter((item)=>item.category==="Podcasts")
+    :type==="Movies"
+    ? arr.filter((item)=>item.category==="Movies")
+    :arr
+}
```
Comment (OmKakatkar) [inline src/Backend/controllers/filter.js:19]: Avoid Long chain of ternary operators

File: src/Pages/VideoDetails/VideoDetails.jsx [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+        .filter((item) => item.id === videoId)
+        .map(({ id, title, source, img }) => (
+          <div key={id} className="video-header-container">
+            <iframe
+              className="video-header"
+              src={`https://www.youtube.com/embed/${id}`}
+              title="YouTube video player"
+              frameBorder="0"
+            //   allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+            //   allowfullscreen
```
Comment (OmKakatkar) [inline src/Pages/VideoDetails/VideoDetails.jsx:21]: Remove commented code

**Review by OmKakatkar — APPROVED** [review]
Like the code structure
Reducers are separated

File: src/Pages/Playlist/PlayList.jsx [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  const { state, dispatch } = useContext(CardContext);
+  return (
+    <div className="recommended-videos color-primary">
+      <div>
+      <h3>Playlist</h3><button onClick={()=>dispatch({type:"ClearPlayList"})} className="login-btn">Clear All</button>
+      </div>  
+      {state.playlists.map(({ id, img, title, source }) => (
+          <Link className="link-tag" to={`/videodetails/${id}`}>
+            <div className="videoCard">
+              <img className="video-thumbnail" src={img} alt="" />
```
Comment (avindhawale) [inline src/Pages/Playlist/PlayList.jsx:17]: Provide value to the alt attribute, it considers as lint error and this helps in screen-reading.

**Review by avindhawale — APPROVED** [review]
Good and simple code structure. Just fix the comment and use mobile first approach in responsive css.

**Pull request merged**